### PR TITLE
allow higher version of i18n

### PR DESCRIPTION
**Pull request by pi3r**
allow higher version of i18n to be installed/loaded. aims to fix the CI for Rails 5

### CODE REVIEW

File: offsite_payments.gemspec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
 
   s.add_dependency('activesupport', '>= 3.2.14', '< 5.1')
-  s.add_dependency('i18n', '~> 0.5')
+  s.add_dependency('i18n', '>= 0.5', '<= 7')
```
Comment (jasonwebster) [inline offsite_payments.gemspec]: do you mean `<= 0.7.0` ?

Comment (pi3r) [inline offsite_payments.gemspec]: @jasonwebster good catch. It doesn't fix the CI though...

**Comment by bdewater** [comment]
You'll want `<= 0.8` because Rails 5 requires `~> 0.7`: https://github.com/rails/rails/blob/5-0-stable/activesupport/activesupport.gemspec#L23

**Comment by bdewater** [comment]
We also need to bump the money gem to ~> 6.0 because via the current ~> 5.0 requirement we're still locking i18n to ~> 0.6.0. https://github.com/RubyMoney/money/blob/master/CHANGELOG.md#600 looks like it shouldn't be too hard.

**Comment by pi3r** [comment]
closed in favour of https://github.com/activemerchant/offsite_payments/pull/217

**Pull request closed without merging**